src/annulus.jl

```diff
@@ -179,7 +179,35 @@ function grid(S::ZernikeAnnulus{T}, B::Block{1}) where T
 
     # The angular grid:
     θ = (0:M-1)*convert(T,2)/M
-    RadialCoordinate.(r, π*θ')
+    RadialCoordinate.(r, convert(T,π)*θ')
+end
+
+_angle(rθ::RadialCoordinate) = rθ.θ
+
+function plotgrid(S::ZernikeAnnulus{T}, B::Block{1}) where T
+    N = Int(B) ÷ 2 + 1  # polynomial degree
+    g = grid(S[:,Block.(OneTo(2N))]) # double sampling
+    θ = [map(_angle,g[1,:]); 0]
+    ρ = S.ρ + eps(T)
```

Comment:
Why `+ eps(T)`? Just to avoid going outside? If so make a comment

Reply:
yes, it sporadically throws an `BoundsError` otherwise. Ok!